setup.py

```diff
@@ -16,7 +16,7 @@
     packages=setuptools.find_packages(),
     install_requires=[
         'requests==2.25.1',
-        'python-dateutil==2.8.1'
```

Comment:
this was originally added for data views, back ~7/3/2019, did we check to make sure removing this works?

Reply:
if i remember correctly we needed it in the requirements but it wasn't actually needed to be imported anywhere to use the functionality we needed.

I forget the exact issue that had come up, but i was working with @Christian Foisy on this then

Reply:
Maybe I'll leave it in to play it safe, then. I assumed since it wasn't imported anywhere that it was just an old import that wasn't used any more.

Reply:
Restored this require